app/controllers/admin/users_controller.rb

```diff
@@ -285,5 +294,24 @@ def send_email_params
       params.require(EMAIL_ALLOWED_PARAMS)
       params.permit(EMAIL_ALLOWED_PARAMS)
     end
+
+    def credit_params
+      credit_params = nil
```

Comment:
Maybe we can restructure this a little bit, something like this:

```ruby
def credit params
  return user_params unless FeatureFlag.enabled?(:admin_member_view)

  credit_params = {}
  if user_params[:credit_action] == "Add"
    ...
  
  credit_params
end
```

Reply:
✅